setup.py

```diff
@@ -1,3 +1,5 @@
+"""Define setup function using setup.cfg."""
```

Comment:
isn't that obvious? :)

Reply:
yes Mr. Obvious, but I can't make pydocstyle ignore this file :D